alpha/lib/model/LiveEntry.php

```diff
@@ -538,14 +537,14 @@ public function getMediaServer($currentDcOnly = false)
 	protected function getMediaServersHostnames()
 	{
 		$hostnames = array();
-		$kMediaServers = $this->getMediaServers();
+		$liveEntryServerNodes = $this->getLiveEntryServerNodes();
 
-		foreach($kMediaServers as $kMediaServer)
+		/* @var LiveEntryServerNode $liveEntryServerNode*/
+		foreach($liveEntryServerNodes as $liveEntryServerNode)
 		{
-			if($kMediaServer instanceof kLiveMediaServer)
-			{
-				$hostnames[$kMediaServer->getIndex()] = $kMediaServer->getHostname();
-			}
+			/* @var WowzaMediaServerNode $serverNode*/
+			$serverNode = ServerNodePeer::retrieveByPK($liveEntryServerNode->getServerNodeId());
+			$hostnames[$serverNode->getType()] = $serverNode->getHostname();
```

Comment:
This is wrong should be:
$hostnames[liveEntryServerNode->getServerType()] = $serverNode->getHostname();